http3/transport.go

```diff
@@ -258,7 +263,10 @@ func (t *Transport) getClient(ctx context.Context, hostname string, onlyCached b
 		go func() {
 			defer close(cl.dialing)
 			defer cancel()
+			traceConnectStart(trace, "udp", hostname)
```

Comment:
Should this distinguish between "udp4" and "udp6"? What does the standard library implementation do? Do they use "tcp", or "tcp4" / "tcp6"?

Reply:
The standard library supports `tcp`, `tcp4`, and `tcp6`; this information is carried down from `net.Dial`. However, quic-go does not follow this approach, so using the default `udp` means it will automatically handle ipv4 and ipv6.